taui/src/components/with-authenticator.js

```diff
@@ -252,7 +284,12 @@ export default function withAuthenticator (Comp, includeGreetings = false,
       } else if (voucher) {
         // attempt to go to client profile directly
         this.goToClientProfile(voucher, email).then(succeeded => {
-          this.setState({authState: state, authData: data})
+          if (succeeded) {
+            this.setState({authState: state, authData: data})
+          } else {
+            console.warn('Failed to load client profile; logging out')
+            this.logoutEcholocator(this.state.authData)
```

Comment:
When I tested out the "invite->delete->invite different->login with old" workflow, this issued the correct message to the console, but I wasn't redirected to the login screen for some reason. I didn't see any console errors, however.

Reply:
How about I make a separate issue for that, so we can unblock design?

Reply:
:+1: Works for me, this is not something that's going to come up frequently.

Reply:
Created #291